### DISCUSSION TITLE
article/notion-next-secondary-menu

### DISCUSSION THREAD

**Discussion by giscus** [tangly1024/tangly1024#67] [Announcements]
# article/notion-next-secondary-menu

NotionNext 3.13.0上线，支持更灵活的菜单配置

https://www.tangly1024.com/article/notion-next-secondary-menu

**Comment by LH1228** [comment]
请问如何让普通菜单点击之后跳转到自己写的文章

**Reply by CATO20230328** [reply]
直接复制你的文章链接到链接栏
